Update faqs.md (#75)
aphrc-dswb/dswb-open-science-capacity-wg: ways-of-working/faqs.md

<details>
<summary><b> Where can Pathfinder members add information on GitHub? </b></summary>
<br>
Each Pathfinder organisation has a GitHub repository created to upload their work to. This can be in the form of meeting reports, OMOP standards, anonymised data, or any other important information that should be public. 
<br>  
<br> 
<strong> Note: </strong> It is important that the information uploaded on GitHub is not sensitive data. Kindly confirm with your Project Coordinator what is considered sensitive if you're unsure.

> <br> You can find the various repositories for each Pathfinder <a href="https://github.com/orgs/aphrc-dswb/repositories"> here </a>.

Reviewer: ```suggestion
<br> Here are the <a href="https://github.com/orgs/aphrc-dswb/repositories">various repositories</a> for each Pathfinder.
```

</details>

<details>
<summary><b> Where can I find records of previous meetings? </b></summary>
<br>

> Previous meetings can be found on the DSWB website, <a href="https://dswb.africa/dswb-events/"> here </a>.

Reviewer: ```suggestion
Previous meetings can be found on the <a href="https://dswb.africa/dswb-events/">DSWB website</a>.
```

</details>

<details>
<summary><b> Can I practice my GitHub learning on our website? </b></summary>
<br>

> Yes, please. We have a repository on our GitHub page for you to play around and test your GitHub knowledge. It is named "dswb-workshops". 

Reviewer: ```suggestion
Yes, please. You can use our <a href="https://github.com/aphrc-dswb/dswb-workshops">dswb-workshops</a> repository to play around and test your GitHub knowledge.
```

<br>  
<br> 

> Feel free to break it, you'd get help! The link is <a href="https://github.com/aphrc-dswb/dswb-workshops"> here </a>. 

Reviewer: ```suggestion
Don't worry if you break anything, we'll fix it!
```
